website/content/docs/concepts/managed-keys.mdx

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+page_title: Managed Keys
+description: >-
+  Managed Keys is a concept in Vault that defers all private key operations to a third party system.
+---
+
+# Managed Keys
+
+~> Note: Managed Keys is an Enterprise only feature.
+
+Within certain environments, customers want additional security guarantees of
+Hardware Security Modules (HSMs) when handling, storing, and interacting with
```

Comment:
I'd be less specific about HSMs which will be just one type, and the cloud ones aren't necessarily hardware backed.  Something about key management systems external to Vault, including HSMs?

Reply:
Addressed, dropped everything about HSMs and just used "to leverage key managed systems external to Vault"